script.js

```diff
@@ -1,4 +1,258 @@
+// 2 players, each player rolls 2 dice,
+// they choose the order of the dice,
+// whoever gets the bigger number wins
+// 2 modes, player 1 and player 2
+// new mode with variable number of dice
+
+var mode = 'enter mode';
+var dice1 = 0;
+var dice2 = 0;
+var myOutputValue = '';
+var playerNumber = 0;
+var player1Number = 0;
+var player2Number = 0;
+
+var playerDiceArray = [];
+var player1DiceArray = [];
+var player2DiceArray = [];
+var dicePositionArray = [];
+var dicePosition1Array = [];
+var dicePosition2Array = [];
+var playerScore = 0;
+var player1Score = 0;
+var player2Score = 0;
+var player1TotalScore = 0;
+var player2TotalScore = 0;
+
+// generate 1 dice number
+var randomNumber = function () {
+  return (Math.floor(Math.random() * 6)) + 1;
+};
+
+// generate 2 dice numbers
+var roll2Dice = function () {
+  var message = '';
+
+  dice1 = randomNumber();
+  dice2 = randomNumber();
+  message = `you rolled<br>
+  Dice 1: ${dice1}<br>
+  Dice 2: ${dice2}<br>`;
+
+  return message;
```

Comment:
Not intuitive that this function called `roll2Dice` returns the message instead of an array of dice numbers